### PR TITLE
[Merged by Bors] - fix: make `to_additive` deal with `abbrev` correctly

### PR DESCRIPTION
**Pull request by astrainfinita**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by eric-wieser** [comment]
Does this change need #15483, or are they really orthogonal?

**Comment by mattrobball** [comment]
@FR-vdash-bot please ping me if the changes to to_additive don't "just build" now

File: Mathlib/Tactic/ToAdditive/Frontend.lean [hunk 1]
```diff
@@ -729,6 +729,16 @@ def findAuxDecls (e : Expr) (pre : Name) : NameSet :=
     else
       l
 
+/-- It's just the same as `Lean.Meta.setInlineAttribute` but with type `CoreM Unit`.
+
+TODO: make `Lean.Meta.setInlineAttribute` a `CoreM Unit` and remove this definition. -/
+def setInlineAttribute (declName : Name) (kind := Compiler.InlineAttributeKind.inline) :
```
Comment (mattrobball) [inline Mathlib/Tactic/ToAdditive/Frontend.lean:735]: Can you make a core issue and add the number as a TODO?

**Comment by mattrobball** [comment]
Can you include a test verifying the new functionality? Thanks

**Comment by mattrobball** [comment]
bors merge

**Pull request closed without merging**